Azure DevOps connector for VIAcode IMS configuration guide.md

```diff
@@ -121,7 +121,7 @@ Press "Submit" button
 
 ## Additional information
 
-If all parts was successfully configured, each time upon delegation a new article will be added to the ticket, which will contain delegation details. Moreover, on each VIMS ticket update, DevOps also will be received related notification:
+Upon delegation a new article will be added to the ticket, which will contain delegation details. Moreover, on each VIMS ticket update, DevOps also will be received related notification:
 
```

Comment:
Here I want to have either link to guide how to use Delegation or step-by-step guide to delegate, validate if PBL created in ADO, move PBL into Done and see article in VIMS. This test delegation should be perfect validation.